app/views/nonprofits/supporters/_manage_tag_definition_modal.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
-# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE -%>
-<div class='modal' id='manageTagMasterModal'>
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE -%>
```

Comment:
This url should be `https://github.com/houdiniproject/houdini/blob/main/LICENSE`